src/sentry/api/endpoints/organization_metrics.py

```diff
@@ -74,10 +59,15 @@ class OrganizationMetricsDetailsEndpoint(OrganizationEndpoint):
     """Get the metadata of all the stored metrics including metric name, available operations and metric unit"""
 
     def get(self, request: Request, organization) -> Response:
-        # TODO: fade out endpoint since the new metrics endpoint will be used.
         projects = self.get_projects(request, organization)
+        if not projects:
+            raise InvalidParams("You must supply at least one project to see its metrics")
+
+        start, end = get_date_range_from_params(request.GET)
```

Comment:
nit: maybe it would be simpler to provide some sensible defaults here in case there were no params provided, so that all of the datasource functions always get `start` and `end`.

Reply:
We have defaults, it's in the params of the function. It's 90 days by default. I decided to keep it like this for consistency with the product.

Reply:
We have defaults, it's in the params of the function. It's 90 days by default. I decided to keep it like this for consistency with the product.